### PR TITLE
Added a newtodo activity

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes(#1)
Also added a menu item in main activity for a new task.
screenshot_20190112_142158

### CODE REVIEW

**Comment by chanmol1999** [comment]
I have updated the pr

File: app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.example.shivansh.to_dolist;
 
+import android.content.Intent;
```
Comment (shiv07tiwari) [inline app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java]: Remove this import statement

**Comment by shiv07tiwari** [comment]
@chanmol1999 Added some suggestions.
Also, when you are done with them, just squash your commits into one. Feel free to ask for  help in this if you have any doubts.
You are doing great ! :smile:

**Comment by shiv07tiwari** [comment]
Also, have you added yourself to participants list ?

**Comment by chanmol1999** [comment]
Updated Don't know about squashing

**Comment by shiv07tiwari** [comment]
@chanmol1999 Why is your PR showing so many irrelevant commits ?
Please remove them
Ask me on slack if you have doubts regarding anything.

**Pull request closed without merging**